src/test/java/seedu/address/testutil/ActivityBuilder.java

```diff
@@ -50,13 +50,15 @@ public ActivityBuilder addPerson(Person person) {
         return this;
     }
 
-    // TODO: Add functionality to add expenses once it is working
     /**
      * Adds an {@code Expense} into the {@code Activity} that we are building.
+     * This is slow and should not be used. Use the addExpense function on the
+     * Activity class directly after creating an Activity.
      */
     public ActivityBuilder addExpense(Expense expense) {
-        expenses.add(expense);
-        return this;
+        Activity temp = this.build();
+        temp.addExpense(expense);
+        return new ActivityBuilder(temp);
     }
```

Comment:
Should we just delete it?

Reply:
We could, the only convenience this provides is chaining.